homeassistant/components/config/entity_registry.py

```diff
@@ -114,6 +115,15 @@ async def websocket_update_entity(hass, connection, msg):
             websocket_api.error_message(msg["id"], "invalid_info", str(err))
         )
     else:
+        result = websocket_api.result_message(msg["id"], _entry_ext_dict(entry))
+        if "disabled_by" in changes and changes["disabled_by"] is None:
+            config_entry = hass.config_entries.async_get_entry(entry.config_entry_id)
+            if entry and not config_entry.supports_unload:
+                result["result"]["requires_restart"] = True
+            else:
+                result["result"][
+                    "reload_delay"
+                ] = config_entries.RELOAD_AFTER_UPDATE_DELAY
```

Comment:
Maybe this should be reworked such that the entity_entry is moved to a sub dict :
```py
result = websocket_api.result_message(msg["id"], {"entity_entry":_entry_ext_dict(entry)})
```

Reply:
Yes, better move the entry to a dict to not conflate keys.

Reply:
OK, fixed.